src/api/app/views/webui2/webui/request/show.html.haml

```diff
@@ -1,3 +1,7 @@
+:ruby
+  @pagetitle = "Show Request #{@number}"
+  @layouttype = 'custom'
```

Comment:
Are we using this?

Reply:
No, it's a remnant from the [old webui](https://github.com/openSUSE/open-build-service/blob/5b51a39e1b448c2116c9e70715b2fe79593e041e/src/api/app/views/layouts/webui/webui.html.erb#L80)

We need to remove it in the other webui2 views. Only a few have this unneeded instance variable.

Reply:
I use it in the Overview. As it is part of the Overview, I suggest to close this PR..

Reply:
I will send a PR to remove the other webui2 views which have this unneeded instance variable.

Reply:
The PR: #6852

Reply:
@krauselukas we can close this PR.